tests/utils/test_serializers.py

```diff
@@ -93,6 +95,31 @@ def serialize_foo(value: Foo) -> str:
     assert not serializers.has_serializer(Foo)
 
 
+class TestEnum(str, Enum):
+    """TestEnum."""
+
+    FOO = "foo"
+    BAR = "bar"
+
+
+class TestEnum2(Enum):
+    """TestEnum2."""
+
+    FOO = "foo"
+    BAR = "bar"
+
+
+@serializers.serializer
+def serialize_TestEnum2(enum: TestEnum2) -> str:
+    return "prefix_" + enum.value
+
+
+class TestBase(Base):
```

Comment:
```suggestion
class BaseSubclass(Base):
```

Again, avoiding the `Test` prefix to not get picked up in pytest collection routine (and avoid warning)

Reply:
![image](https://github.com/reflex-dev/reflex/assets/1524005/19b920e4-a86a-46e3-a002-49202c692f53)

Reply:
So THAT was the reason for this warnings, thanks!